### PR TITLE
PR for Addition of components for display of buttons and routes. 

### PR DESCRIPTION
**Pull request by CrystallineButterfly**
Hi there, 

This is a PR, to show the basic use of functional components to enable a basic interface with buttons for adding and removing books, 
with no functionality yet, just to show use cases of routes and components in react, in prep for an upcoming project. 

Thank you for the code review!

All the best-K

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
+          <Link to="/catergories">Catergories</Link>
+        </li>
+      </ul>
+      <hr />
+      <Routes>
+        <Route exact path="/books" element={<DisplayBooks />} />
+        <Route path="/catergories" element={<Catergories />} />
+      </Routes>
+      <AddBooks />
+      <RemoveBook />
```
Comment (brytebee) [inline src/App.js]: - [x] **Required:** These two components should be rendered on the `DisplayBooks` component and should only be visible on the books page.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @CrystallineButterfly👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- The project is initialized using create-react-app.✔️
- `react-router-dom` is present in package.json. It is used to make the routes.✔️
- Components are well organized and rendered.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️

- Update the title of your README. This makes your documentation more professional.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (CrystallineButterfly) [reply src/App.js]: Hi, @brytebee Thank you for the review, however, that is not a requirement in this project, so please explain what makes this more efficient to do, more than my current way of doing it? 

All the best-K

Comment (RolandM99) [reply src/App.js]: - [x] As requested by the previous reviewer, make sure that you render those two components in the `DisplayBooks` component. For now, we can see the form for adding and removing books on the `home` page and on the `categories` page but, it should only see on the homepage. 
- To implement this change you can only remove those two components from your `App.js` component, which means from this file with the function Bookstore, and then add them in the `DisplayBooks.js` component like in the screenshot below:
image

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @CrystallineButterfly, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CrystallineButterfly** [comment]
Hi, @RolandM99 Ok thank you for the in-depth explanation I have implemented the change now!

All the best-K

**Review by Whoistolu — APPROVED** [review]
### Hi @CrystallineButterfly 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**